python/aibrix_kvcache/aibrix_kvcache/envs.py

```diff
@@ -345,6 +349,16 @@
             "AIBRIX_KV_CACHE_OL_TRANSPORT_RDMA_ADDR_RANGE", "::/0"
         ).strip()
     ),
+    # ==================== Profiling Env Vars ====================
+    "AIBRIX_KV_CACHE_OL_PROFILING_ENABLED": lambda: (
+        os.getenv("AIBRIX_KV_CACHE_OL_PROFILING_ENABLED", "0").strip().lower()
+        in ("1", "true")
+    ),
+    "AIBRIX_KV_CACHE_OL_PROFILING_SERVER_ADDRESS": lambda: (
+        os.getenv(
+            "AIBRIX_KV_CACHE_OL_PROFILING_SERVER_ADDRESS", "http://0.0.0.0:4040"
```

Comment:
what's the best practice here? we should run pyroscope server locally and run aibrix kvcache + inference server?

If we enable online profiling, we need a cluster ip? 

It would be great to write some docs on the practice